docs/source/dev-setup/devenv.rst

```diff
@@ -46,6 +46,125 @@ Prerequisites
 
     brew install jq
 
+Go Programming Language
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Hyperledger Fabric uses the Go Programming Language for many of its
+components.
+
+  - `Go <https://golang.org/dl/>`__ version 1.14.x is required.
+
+There are two environment variables you will need to set properly; you can make these settings permanent by placing them in the appropriate startup file, such as your personal ``~/.bashrc`` file if you are using the ``bash`` shell under Linux.
+
+First, you must set the environment variable ``GOPATH`` to point at the
+Go workspace containing the downloaded Fabric code base, with something like:
+
+.. code:: bash
+  export GOPATH=$HOME/go
+.. note:: You **must** set the GOPATH variable
+
+  Even though, in Linux, Go's ``GOPATH`` variable can be a colon-separated list
+  of directories, and will use a default value of ``$HOME/go`` if it is unset,
+  the current Fabric build framework still requires you to set and export that
+  variable, and it must contain **only** the single directory name for your Go
+  workspace. (This restriction might be removed in a future release.)
+
+Second, you should (again, in the appropriate startup file) extend your
+command search path to include the Go ``bin`` directory, such as the following
+example for ``bash`` under Linux:
+
+.. code:: bash
+  export PATH=$PATH:$GOPATH/bin
```

Comment:
And this is not quite correct. The tools are currently installed into the [default GOBIN directory](https://github.com/hyperledger/fabric/blob/6626a86d027538ec5abda834230525dab4de70f3/gotools.mk#L8). If you have explicitly set the `GOBIN` environment variable, that's where the tools will be installed; if not, go will use the equivalent of `$(go env GOPATH)/bin`. If `GOPATH` isn't set in the environment, it defaults to $HOME/go on *nix.

So, the GOPATH reference is unnecessary and if someone simply clones into any directory, things should be fine.

Reply:
removed